submissions/victorAienobe/htmlForms/contactForm.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Contact Form</title>
+    </head>
+    <body>
+        <form action="#" method="POST" enctype="multipart/form-data">
```

Comment:
@ainobs Remove attributes `action`, `method` and `enctype`, they are not needed yet.